tests/unit/pages/SplashPage.spec.js

```diff
@@ -161,5 +162,24 @@ describe('Test component: SplashPage', () => {
       expect(initUserRoles).toHaveBeenCalledTimes(0);
       expect(Notify.create).not.toHaveBeenCalledWith(expect.objectContaining({ type: 'negative' }));
     });
+
+    it('should redo login if session token is expired', async () => {
+      getUserSessionToken.mockImplementation(() => true);
+      initUserInformation.mockImplementation(() => Promise.reject(
+        {
+          response: {
+            data: { code: 209 },
+          },
+        },
+      ));
+      setActivePinia(createPinia());
+      Notify.create = jest.fn();
+
+      await wrapper.vm.initUser();
+
+      expect(getUserSessionToken).toHaveBeenCalledTimes(4);
```

Comment:
Why four times ?

Reply:
Because if everything went well the getUserSessionToken is called 4 times